eng/Version.Details.xml

```diff
@@ -214,5 +214,9 @@
       <Uri>https://github.com/dotnet/hotreload-utils</Uri>
       <Sha>25b814e010cd4796cedfbcce72a274c26928f496</Sha>
     </Dependency>
+    <Dependency Name="System.Runtime.Numerics.TestData" Version="6.0.0-beta.21314.1">
+      <Uri>https://github.com/dotnet/runtime-assets
```

Comment:
Just to let you know, this broke dependency flow as you were missing the `</Uri>` element. I'm fixing this in https://github.com/dotnet/runtime/pull/54511.

Reply:
Ah sorry my bad. Thanks for fixing it!